EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -377,6 +389,41 @@ void EarthAsDMAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup&
     bool isTrackerHighPtMuon = muon::isTrackerHighPtMuon(*muon, highestSumPt2Vertex);
     muon_isTrackerHighPtMuon_.push_back( isTrackerHighPtMuon);
 
+    // Match muon inner tracks with general track
+    bool hasMatchedGenTrack = 0;
+    float genTrackPt = 9999.; 
+    float genTrackPtErr = 9999.; 
+    float genTrackEta = 9999.;
+    float genTrackPhi = 9999.;
```

Comment:
these are correct, but if you want to be consistent with the defaults, maybe they should be put to negative too (this is not a must tho)